### ISSUE TITLE
rewrite in Windows UWP

### ISSUE THREAD

**Issue by Inderix** [twitter/the-algorithm#2068]

**Comment by dumbButSkilledDev** [comment]
it is time....

**Comment by PatoFlamejanteTV** [comment]
Agreed. (btw I also have a Windows Phone and I need help sideloading apps since the Microsoft deleted the app store from Windows Phones)

**Comment by DennisHerbJohnson** [comment]
@Inderix INDERIX!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by DennisHerbJohnson** [comment]
@Inderix 

<img width="339" height="320" alt="INDERIX!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!" src="https://github.com/user-attachments/assets/f5ac0691-6790-4852-864c-7e5982c3b9fd" />
